### PR TITLE
Upgrade: define new release repos and remove old

### PR DESCRIPTION
**Pull request by mnasiadka**

### CODE REVIEW

**Comment by cityofships** [comment]
Needs role's README update with new vars

**Comment by mnasiadka** [comment]
> Needs role's README update with new vars, otherwise LGTM.

Well, I don't think those variables should be changed by user - but then probably we should move them to vars/ instead of defaults/.

**Review by cityofships — APPROVED** [review]

File: roles/cephadm/tasks/pkg_debian.yml [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
+    repo: "deb https://download.ceph.com/debian-{{ item }}/ {{ ansible_distribution_release }} main"
+    state: "{{ 'present' if item == cephadm_ceph_release else 'absent' }}"
   when: not cephadm_custom_repos | bool
   become: true
+  loop: "{{ cephadm_ceph_versions }}"
 
 - name: Install cephadm package
   apt:
     name: "cephadm"
     state: "{{ 'latest' if cephadm_package_update | bool else 'present' }}"
```
Comment (markgoddard) [inline roles/cephadm/tasks/pkg_debian.yml:19]: Could end up in a weird state if you change cephadm_ceph_release but don't set cephadm_package_update.

File: roles/cephadm/vars/main.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+cephadm_rpm_repos:
+  - "$basearch"
+  - "noarch"
+cephadm_ceph_versions:
```
Comment (markgoddard) [inline roles/cephadm/vars/main.yml:5]: nit: releases?

Comment (mnasiadka) [reply roles/cephadm/vars/main.yml:5]: pushed a PR since it's a trivial fix

File: roles/cephadm/tasks/pkg_debian.yml (hunk 1 above)
Comment (mnasiadka) [reply roles/cephadm/tasks/pkg_debian.yml:19]: Well, yes - let me fix it.

**Pull request merged**